service/worker/batcher/activities.go

```diff
@@ -67,7 +67,12 @@ func (a *activities) checkNamespace(namespace string) error {
 func (a *activities) BatchActivity(ctx context.Context, batchParams BatchParams) (HeartBeatDetails, error) {
 	logger := a.getActivityLogger(ctx)
 	hbd := HeartBeatDetails{}
+	activityInfo := activity.GetInfo(ctx)
+	metricsClient := a.MetricsClient.Scope(metrics.BatcherScope, metrics.NamespaceTag(activityInfo.WorkflowNamespace))
```

Comment:
why not just use `batchParams.Namespace`?

Reply:
Updated